SimG4Core/Application/python/g4SimHits_cfi.py

```diff
@@ -559,3 +562,5 @@
 phase2_timing.toModify( g4SimHits.ECalSD,
                              StoreLayerTimeSim = cms.untracked.bool(True),
                              TimeSliceUnit = cms.double(0.001) )
+from Configuration.Eras.Modifier_phase2_common_cff import phase2_common
+phase2_common.toModify( g4SimHits.CaloTrkProcessing, DoFineCalo = True)
```

Comment:
Do we want this enabled by default for Phase2? Is there a change in processing time or event content size?

Reply:
Initially I thought of having very high threshold and have it as default for phase2. But thinking again, I see your point - it may be better to have reasonable threshold and remove this from default so that only interested people will use this

Reply:
@bsunanda , @kpedro88 , I agree that this is for detector study and not for production.